src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="todoapp__main">
+    {todos.map((todo) => (
+      <div
+        className="todo"
+        key={todo.id}
+      >
+        <label className="todo__status-label">
+          <input
+            type="checkbox"
+            className="todo__status"
+          />
+        </label>
+
+        <span className="todo__title">{todo.title}</span>
+        <button type="button" className="todo__remove">×</button>
+
+        <div className="modal overlay">
+          <div
+            className="modal-background has-background-white-ter"
+          />
+          <div className="loader" />
+        </div>
+      </div>
+    ))}
+    {
+      // #region todosDiffTypes
```

Comment:
Please remove commented out code and unused examples, as they make the code less readable.

Reply:
agree